akka-stream/src/main/scala/akka/stream/javadsl/Graph.scala

```diff
@@ -278,10 +277,10 @@ object FlowGraph {
    * The [[Builder]] is mutable and not thread-safe,
    * thus you should construct your Graph and then share the constructed immutable [[FlowGraph]].
    */
-  def builder(): Builder = new Builder()(new scaladsl.FlowGraph.Builder)
+  def builder[Mat](): Builder[Mat] = new Builder()(new scaladsl.FlowGraph.Builder[Mat])
```

Comment:
TBH it doesn't tell me anything about what it is—it's even worse because its a real word:

`a piece of coarse material placed on a floor for people to wipe their feet on.`